src/main/kotlin/no/nav/familie/ks/sak/kjerne/eøs/kompetanse/KompetanseService.kt

```diff
@@ -90,67 +93,99 @@ class KompetanseService(
         tilBehandlingId: BehandlingId,
     ) =
         kompetanseSkjemaService.kopierOgErstattSkjemaer(fraBehandlingId, tilBehandlingId)
+}
 
-    private fun tilpassKompetanserTilRegelverk(
-        eksisterendeKompetanser: Collection<Kompetanse>,
-        barnaRegelverkTidslinjer: Map<Aktør, Tidslinje<RegelverkResultat>>,
-        barnasHarEtterbetaling3MånedTidslinjer: Map<Aktør, Tidslinje<Boolean>>,
-        annenForelderOmfattetAvNorskLovgivningTidslinje: Tidslinje<Boolean>,
-    ): List<Kompetanse> {
-        val barnasEøsRegelverkTidslinjer =
-            barnaRegelverkTidslinjer
-                .tilBarnasEøsRegelverkTidslinjer()
-                .leftJoin(barnasHarEtterbetaling3MånedTidslinjer) { regelverk, harEtterbetaling3Måned ->
-                    when (harEtterbetaling3Måned) {
-                        true -> null // ta bort regelverk hvis barnet har etterbetaling 3 måned
-                        else -> regelverk
-                    }
-                }
+@Service
+class TilpassKompetanserTilEndretUtbetalingAndelerService(
```

Comment:
Kunne man flytta ut denne her? 
Synes det blir litt trangt med to services i en fil